cmake/Modules/FindAtlas.cmake

```diff
@@ -23,14 +23,14 @@ set(Atlas_LIB_SEARCH_PATHS
   $ENV{Atlas_ROOT_DIR}/lib
 )
 
-find_path(Atlas_CBLAS_INCLUDE_DIR NAMES cblas.h PATHS ${Atlas_INCLUDE_SEARCH_PATHS})
+find_path(Atlas_CBLAS_INCLUDE_DIR   NAMES cblas.h   PATHS ${Atlas_INCLUDE_SEARCH_PATHS})
 find_path(Atlas_CLAPACK_INCLUDE_DIR NAMES clapack.h PATHS ${Atlas_INCLUDE_SEARCH_PATHS})
-find_library(Atlas_CBLAS_LIBRARY NAMES ptcblas_r ptcblas cblas_r cblas PATHS ${Atlas_LIB_SEARCH_PATHS})
-find_library(Atlas_BLAS_LIBRARY NAMES atlas_r atlas PATHS ${Atlas_LIB_SEARCH_PATHS})
-find_library(Atlas_LAPACK_LIBRARY NAMES alapack_r alapack lapack_atlas PATHS ${Atlas_LIB_SEARCH_PATHS})
 
-set(LOOKED_FOR
+find_library(Atlas_CBLAS_LIBRARY NAMES  ptcblas_r ptcblas cblas_r cblas PATHS ${Atlas_LIB_SEARCH_PATHS})
+find_library(Atlas_BLAS_LIBRARY NAMES   atlas_r   atlas                 PATHS ${Atlas_LIB_SEARCH_PATHS})
+find_library(Atlas_LAPACK_LIBRARY NAMES alapack_r alapack lapack_atlas  PATHS ${Atlas_LIB_SEARCH_PATHS})
```

Comment:
Actually I don't know whether it is a typo or not. I copied these lines from a file from previous version of caffe cmake scrips (from [here](https://github.com/BlGene/caffe/blob/ececfc0c0ca98e242ef3b7634f1bfcb431028bd8/CMakeScripts/FindAtlas.cmake#L26-L30))

I don't know if on a OS it is not called as 'alapack'. So the best solution is just to write the following:

```cmake
+find_library(Atlas_LAPACK_LIBRARY NAMES alapack_r 
           alapack lapack_atlas lapack PATHS ${Atlas_LIB_SEARCH_PATHS})
```
i.e. just add `lapack` to the libraries search list.

@boaz001 Would you mind creating PR? Or I can do this.